Makefile

```diff
@@ -25,9 +25,16 @@ run-ios: .npminstall config/config.secret.json
 		echo "xcode is not installed"; \
 		exit 1; \
 	fi
+	@if ! [ $(shell command -v watchman) ]; then \
+		echo "watchman is not installed"; \
+		exit 1; \
+	fi
+
 	@echo Running iOS app in development
 
 	npm run run-ios
+	open -a Simulator
```

Comment:
Won't `npm run run-ios` already open the simulator? Is the purpose of this added line to trigger an obvious failure if the simulator isn't already installed? If so, let's fire up the simulator before we call `run-ios`.